cli/src/main.rs

```diff
@@ -216,12 +216,19 @@ fn override_configuration(mut config: Config, options: &ArgMatches) -> Config {
     }
 
     #[cfg(feature = "go")]
-    if let Some(go_package) = options.value_of(args::ARG_GO_PACKAGE) {
-        config.go.package = go_package.to_string();
+    {
+        match options.value_of(args::ARG_GO_PACKAGE) {
+            Some(go_package) => {
+                config.go.package = go_package.to_string();
+            }
+            None => {
+                config.go.package = is_go_package_present(&config)?;
```

Comment:
Setting this value is a little redundant because we are just reading from the config and then setting it again. I think it would make more sense to just do nothing on this line and then assert that the package exists